service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsSubmittedControllerTest.java

```diff
@@ -288,19 +309,22 @@ private CaseDetails createCase(Map<String, Object> data) {
         );
     }
 
-
     private Map<String, Object> expectedCtscNotificationParameters() {
         return Map.of("applicationType", "C2",
             "caseUrl", "http://fake-url/cases/case-details/12345");
     }
 
-    private Map<String, Object> expectedNotificationParams() {
-        return Map.of(
-            "reference", CASE_ID.toString(),
-            "hearingDetailsCallout", String.format("%s, %s", RESPONDENT_SURNAME, CASE_ID.toString()),
-            "subjectLine", String.format("%s, %s", RESPONDENT_SURNAME, CASE_ID.toString()),
-            "caseUrl", "http://fake-url/cases/case-details/" + CASE_ID
-        );
+    private C2UploadedTemplate expectedNotificationParams() {
```

Comment:
minor: could be `getExpectedNotificationParams()`

Reply:
changed :)